src/sentry/integrations/discord/client.py

```diff
@@ -47,7 +47,7 @@ def with_base_url(self, url: str) -> str:
 
     def overwrite_application_commands(self, commands: list[object]) -> None:
         self.put(
-            self.with_base_url(APPLICATION_COMMANDS_URL.format(application_id=self.application_id)),
```

Comment:
@17hogeju this doesn't impact tests?

Reply:
There was not already a test for this specific line change and none was added when @dashed made the update https://github.com/getsentry/sentry/pull/59594